cmd/query/app/fixture/index.html

```diff
@@ -2,4 +2,5 @@
 <html lang="en">
     <meta charset="UTF-8">
     <title>Test Page</title>
+    <!-- JAEGER_CONFIG=DEFAULT_CONFIG; -->
```

Comment:
change this to: "JAEGER_CONFIG = DEFAULT_CONFIG;" to be a more accurate fixture?

Reply:
how is it more accurate?  The regexp ignores the whitespace.